README.md

```diff
@@ -1,2 +1,38 @@
-# hello-microverse
- My first Microverse Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> My first Microverse Project.
+
+## Built With
+
+- Major languages - HTML & CSS
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://walterokumu.github.io/hello-microverse)
+
+## Getting Started
+
```

Comment:
- [ ] [optional] It would be nice to add how to clone and run the project locally.